rdt/transformers/datetime.py

```diff
@@ -102,9 +102,6 @@ def _reverse_transform_helper(self, data):
         if self.missing_value_replacement is not None:
             data = self.null_transformer.reverse_transform(data)
 
-        if isinstance(data, np.ndarray) and (data.ndim == 2):
-            data = data[:, 0]
```

Comment:
I believe this is from this [PR](https://github.com/sdv-dev/RDT/pull/161/files)
I'm not sure why we would ever need this functionality. The transformer should only get more than one column in the case of nulls which is handled above